app/app.rb

```diff
@@ -293,14 +294,27 @@ def process_form_data(data, git)
 					file.relative_path.to_s
 				elsif data.keys.include?('___checkbox_unchecked')
 					data.keys.include?('___checkbox_checked')
+				elsif data.keys.include?("___datetime_date")
+					datetime = data.values.join(" ")
+					Time.parse(datetime) if datetime.present?
```

Comment:
I prefer typing `datetime.present?` over `datetime.strip != ""`, as it has the added bonus that it works on a bunch of other objects, like Hash and Array. But I would be fine using the `datetime.strip != ""` here if we don't want to import ActiveSupport. Mostly my thinking was that it was installed anyway, so it wouldn't hurt much to use it.